src/test/java/sdmExam/BoardShould.java

```diff
@@ -174,18 +166,19 @@ public void fillTerritoryWithEqualNumberOfStoneOfTheSameColor() {
     public void fillTerritoryWithDifferentNumberOfStone() {
         int boardSize = 13;
         Board customBoard = Board.buildTestBoard(boardSize);
-        IntStream.rangeClosed(1, 6)
-                .forEach(column -> customBoard.addStoneAt(Stone.WHITE, in(7, column)));
-        IntStream.rangeClosed(7, boardSize)
-                .forEach(column -> customBoard.addStoneAt(Stone.BLACK, in(7, column)));
-        IntStream.rangeClosed(1, 4)
-                .forEach(column -> customBoard.addStoneAt(Stone.WHITE, in(9, column)));
-        IntStream.rangeClosed(5, boardSize)
-                .forEach(column -> customBoard.addStoneAt(Stone.BLACK, in(9, column)));
-
-        Stone lastPlay = Stone.BLACK;
-
-        customBoard.fillTerritory(customBoard.getTerritories().get(0), lastPlay);
+        IntStream.rangeClosed(1,boardSize)
+                .forEach(column -> {
+                    if (column <= 6) customBoard.addStoneAt(Stone.WHITE, in(7, column));
```

Comment:
Add curly braces in if-else statements, it leads to fewer bugs.

Reply:
Done